composer.json

```diff
@@ -19,14 +19,15 @@
         }
     ],
     "require": {
-        "php":                     ">=7.0",
+        "php":                     ">=7.2",
```

Comment:
@andrewnicols are you okay with this? I want to limit the number of PHP versions to support.

Reply:
We support PHP 7.0 + 7.1 in Moodle for branches under LTS/security support until May but we were already talking about not migrating from instaclick to php-webdriver for those supported releases due to PHP release status of dependencies.

if you'd prefer to drop support that should be fine. We can always roll our own version if we need to.

Reply:
Thanks for your feedback. I this case I'd like to drop 7.0 and 7.1.